cypress/integration/computation_menu.spec.ts

```diff
@@ -36,7 +36,6 @@ function waitForLoad(iteration: number = 0) {
 
 describe("ComputationMenu", () => {
     beforeEach(() => {
-        cy.clearLocalStorage();
```

Comment:
Should we perhaps clear once per test file?

Reply:
Cypress automatically clears local storage between each test. The command is superfluous unless you need to clear storage in the middle of a test.

Reply:
Each test category? Or each test, as in 860 times?

Reply:
Each test. I have pushed a commit that implements caching and restoring of local storage, to reduce the number of API calls to every 30 seconds, but without slowing down the testing.